tests/plugins/test_koji_import.py

```diff
@@ -605,10 +605,10 @@ def custom_get(method, url, headers, **kwargs):
 def workflow(workflow):
     """Add additional data to provide pipeline_run_name specifically."""
     workflow.user_params.update({
-        'namespace': NAMESPACE,
         'pipeline_run_name': PIPELINE_RUN_NAME,
         'koji_task_id': MockedClientSession.TAG_TASK_ID,
     })
+    workflow.namespace = NAMESPACE
```

Comment:
workflow fixture already has `namespace`.

Reply:
you're right :). Updated.